test/e2e/overrides_test.go

```diff
@@ -41,7 +41,27 @@ var _ = Describe("Overrides test", Ordered, func() {
 		It("should add the args to the cert-manager controller deployment", func() {
 
 			By("Adding cert-manager controller override args to the cert-managaer operator object")
-			args := []string{"--dns01-recursive-nameservers=10.10.10.10:53", "--dns01-recursive-nameservers-only", "--enable-certificate-owner-ref", "--v=3"}
+			args := []string{
+				// good-have to sync these args updated with the args present in
+				// pkg/controller/deployment/deployment_overrides_validation.go,
+				// so the e2e is self-aware of overrideArgs.
+
+				"--acme-http01-solver-resource-limits-cpu=150m",
+				"--acme-http01-solver-resource-limits-memory=100Mi",
+				"--acme-http01-solver-resource-request-cpu=50m",
+				"--acme-http01-solver-resource-request-memory=100Mi",
+
+				"--dns01-recursive-nameservers=10.10.10.10:53",
+				"--dns01-recursive-nameservers-only",
+
+				"--enable-certificate-owner-ref",
+
+				"--issuer-ambient-credentials",
+
+				"--v=5",
+
+				"--metrics-listen-address=0.0.0.0:9403",
```

Comment:
Port 9403 conflicts with an existing cert-manager-controller port `http-healthz` https://github.com/openshift/cert-manager-operator/blob/b6b2d69c98c31e9772546f5b9f7aff5efb99e725/bindata/cert-manager-deployment/controller/cert-manager-deployment.yaml#L63-L65

```
I0721 10:15:13.318951       1 controller.go:102] "listening for insecure connections" logger="cert-manager.controller" address="0.0.0.0:9403"
E0721 10:15:13.319411       1 main.go:40] "error executing command" err="failed to listen on healthz address 0.0.0.0:9403: listen tcp 0.0.0.0:9403: bind: address already in use" logger="cert-manager"
```

Reply:
Updated to 9401, thanks @lunarwhite for bringing this up!